seerep-ros/seerep_ros_conversions_fb/test/ros_to_fb_conversion_test.cpp

```diff
@@ -405,57 +432,43 @@ class rosToFbConversionTest : public testing::Test
         seerep_ros_conversions_fb::toROS(*fb_transform_stamped.GetRoot());
     // Transform End
   }
-};
 
-// Objects created below are used by the TEST functions
+  /**
+   * @brief This function instantiates all the attributes and fills them with arbitrary values.
+   * */
+  void TestBody()
```

Comment:
Do not use this function according to the gTest documentation. Can you not better call those functions in the constructor?

>   // Runs the test after the test fixture has been set up.
  //
  // A sub-class must implement this to define the test logic.
  //
  // DO NOT OVERRIDE THIS FUNCTION DIRECTLY IN A USER PROGRAM.
  // Instead, use the TEST or TEST_F macro.
  virtual void TestBody() = 0;

Reply:
Maybe SetUpTestSuite() is the correct methode?